src/librustc_traits/lowering.rs

```diff
@@ -58,6 +58,8 @@ impl<'tcx> Lower<DomainGoal<'tcx>> for ty::TypeOutlivesPredicate<'tcx> {
     }
 }
 
+/// Lowers a trait or projection predicate to a `FromEnv(...)` clause.
```

Comment:
What does this comment refer to actually?

Reply:
Nothing, I added it by accident!